src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiEdmTypeSchemaGenerator.cs

```diff
@@ -420,7 +420,6 @@ private static OpenApiSchema CreateEnumTypeSchema(this ODataContext context, IEd
             Debug.Assert(typeReference != null);
 
             OpenApiSchema schema = new OpenApiSchema();
-            schema.Nullable = typeReference.IsNullable;
             schema.Reference = null;
 
             if (context.Settings.OpenApiSpecVersion >= OpenApiSpecVersion.OpenApi3_0)
```

Comment:
I'm not sure why the `typeReference.IsNullable` check wasn't done here as in one other place. Why did we have AnyOf when there's only one item in the list?

Reply:
Agreed, I'd argue we should have an any of only if nullable is true, otherwise we're just bloating the description.